### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by arshidkv12**

### CODE REVIEW

**Comment by pwkip** [comment]
Thanks for this PR @arshidkv12 . I see you did a bit of cleanup, but could you explain in a couple of words what changes you made, and why? Did the code trigger any notices/errors before these changes?

**Comment by arshidkv12** [comment]
Please check related comments.

File: admin.php [hunk 1]
```diff
@@ -177,7 +177,7 @@ function wpcf7cf_after_save($contact_form) {
 function wpcf7cf_save_contact_form( $contact_form )
 {
 
-	if ( ! isset( $_POST ) || empty( $_POST ) || ! isset( $_POST['wpcf7cf-settings-text'] ) ) {
```
Comment (pwkip) [inline admin.php:180]: Need to keep this for usage in end to end tests and when calling methods directly from command line.

File: wpcf7cf-options.php [hunk 2]
```diff
@@ -205,8 +205,6 @@ function wpcf7cf_input_field($slug, $args) {
     $args = wp_parse_args( $args, $defaults );
     extract($args);
 
-    $label; $description; $default; $label_editable;
```
Comment (pwkip) [inline wpcf7cf-options.php:208]: keeping this as a helper for development.

**Review by pwkip — APPROVED** [review]

**Pull request merged**